### ISSUE TITLE
Support for static / arbitrary html content

### ISSUE THREAD

**Issue by gowerc** [rstudio/rsconnect-python#182]
Hi,

We are currently trying to use the rsconnect python CLI tool to upload content to rsconnect which is working fine. The only issue is that it can only create manifests for python-specific content. We often use the rsconnect server to host static html / pdf / txt documents (i.e. not Rmd / Shiny) which the CLI cannot create a manifest for forcing us to drop into R.

It would be awesome if the CLI could natively handle this. To give some idea here is an example deployment script we use:

```
export RS_TOKEN=abcdef....

rsconnect add \
    --name apollo\
    --server https://connect.sever.com/ \
    --api-key $RS_TOKEN

## Create Manifest
Rscript -e 'rsconnect::writeManifest(
    appDir = getwd(),
    appFiles = c("index.html", list.files("outputs", recursive = TRUE, full.names = TRUE)),
    appPrimaryDoc = "index.html",
    contentCategory = "site"
)'

rsconnect deploy manifest \
    -t PHC123 \
    -n apollo \
    manifest.json

```

A related issue though (happy to make a second issue if needed) is that launching content this way stores the metadata  under `rsconnect-python/` where as if we were to launch the exact same content from Rstudio GUI the metadata is stored under `rsconnect/` meaning that we end up with 2 separate instances for the same content. It would be awesome if the metadata storage could be unified.

**Comment by gowerc** [comment]
I guess more generally, it would be awesome if this tool could be used complete CLI for all things RSconnect rather than just it being python content focused. This CLI is invaluable for command line deployments such as in CICD jobs

**Comment by edavidaja** [comment]
addressed in #236

**Comment by bcwu** [comment]
Note: We added the ability to deploy html. The feature to deploy text and pdf is most definitely doable, but in my opinion the bottleneck is more on organizing the content once uploaded. we are electing to not add deploy txt/pdf for now, until the organizing UI/UX side of things can collate and curate mass txt/pdfs in good order.
